Kitodo-Docket/src/main/java/org/kitodo/docket/Docket.java

```diff
@@ -43,9 +48,14 @@ public File generateMultipleDockets(Collection<DocketData> docketData, URI xslFi
         File file = new File("docket_multipage.pdf");
         FileOutputStream fileOutputStream = new FileOutputStream(file);
 
-        exportDocket.startExport(docketData, fileOutputStream, new File(xslFileUri));
-        fileOutputStream.flush();
-        fileOutputStream.close();
+        try {
```

Comment:
Please use [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) statement instead of handling this manually.